ar-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+artifact_registry_location=us-central1
+project=$(gcloud config get-value core/project)
+
+usage () {
+    echo "usage: ./ar-setup.sh --app-name <app_name> --gitlab-access-token <token> --app-config-repo <app-config-repo> [--project <gcp_project>] [--artifact-registry-location <location>]"
```

Comment:
I'm thinking users may not know where to find the existing gitlab-access-token. Can we change the flag --gitlab-access-token to be optional? If not set, this script can search for the token in the ~/.appctl/.appctlconfig

Reply:
This only works if the users chose to cache their access tokens in `~/.appctl/.appctlconfig`. 
In the case where `~/.appctl/.appctlconfig` does not include the access tokens, the users still needs to provide the option.

Considering the fact that this script is run after `appctl init`, which already asks the users for their access tokens, we can ask the tokens here again.

We can definitely do more complex logic: if the users do not provide this option, we first check whether `~/.appctl/.appctlconfig` includes a token. If not, tell the users to set the option. However, I am not sure the effort is worthy at this stage. After the demo, if we decide this is the right production direction, we can improve it. WDYT?

Reply:
That sounds reasonable. We can improve it after the demo.